docs/application/web/api/5.0/device_api/wearable/tizen/feedback.html

```diff
@@ -417,88 +411,95 @@ <h4>Methods</h4>
 </li></ul>
         </div>
 <div class="example">
-<span class="example"><p>Code example:</p></span><pre name="code" class="examplecode prettyprint">
-var pattern = "BT_CONNECTED", type = "TYPE_SOUND";
+<span class="example"><p>Code example:</p></span><pre name="code" class="examplecode prettyprint">var pattern = "BT_CONNECTED", type = "TYPE_SOUND";
 var isPatternSupported = tizen.feedback.isPatternSupported(pattern, type);
 var isSupported = "";
 if (!isPatternSupported)
 {
-   isSupported = " not";
+  isSupported = " not";
 }
-console.log("pattern " + pattern + " is" + isSupported + " supported");
+console.log("Pattern " + pattern + " is" + isSupported + " supported");
 </pre>
 </div>
 <div class="output">
-<span class="title"><p>Output example:</p></span><pre> pattern BT_CONNECTED is supported
- </pre>
+<span class="title"><p>Output example:</p></span><pre>Pattern BT_CONNECTED is supported
+</pre>
 </div>
 </dd>
 </dl>
 </div>
 </div>
 </div>
-<h2 id="full-webidl">3. Full WebIDL</h2>
+<h2 id="api-features">3. Related Feature</h2>
+<div id="def-api-features" class="def-api-features">
+        You can check if this API is supported with <em>tizen.systeminfo.getCapability()</em> and decide enable/disable codes that need this API.
+                    <div class="def-api-feature">
+<p><div class="description">
+            <p>
+To guarantee the running of the application on a device which supports feedback vibration feature, since 5.0 declare the following feature requirement in the config file:
```

Comment:
I think, "since Tizen 5.0" would be better. Please, consider adding "Tizen" word.